Library/ViewModels/PledgeViewModelTests.swift

```diff
@@ -1453,42 +1575,46 @@ final class PledgeViewModelTests: TestCase {
       with: (amount: 690, min: 25.0, max: 10_000.0, isValid: true)
     )
 
+    self.submitButtonEnabled.assertValues([false, false], "Amount unchanged")
+
     self.vm.inputs.shippingRuleSelected(.init(cost: 1, id: 1, location: .brooklyn))
 
-    self.submitButtonEnabled.assertValues([false, false], "Shipping rule and amount unchanged")
+    self.submitButtonEnabled.assertValues([false, false, false], "Shipping rule and amount unchanged")
```

Comment:
Should we consider throwing a `skipRepeats()` on this output? 😅

Reply:
Maybe, but not right now 😅